auth-web/src/views/auth/TermsOfServiceView.vue

```diff
@@ -1,52 +1,41 @@
 <template>
-  <v-container class="view-container">
+  <v-container class="terms-of-use-container view-container">
 
     <!-- Loading status -->
     <v-fade-transition>
       <div class="loading-container" v-if="isLoading">
         <v-progress-circular size="50" width="5" color="primary" :indeterminate="isLoading"/>
       </div>
     </v-fade-transition>
-    <div class="" v-if="!isLoading">
-      <v-row justify="center">
-        <v-col lg="8" class="pt-0 pb-0">
-          <div class="view-header">
-            <h1>Terms of Use</h1>
-          </div>
-          <v-card class="profile-card" flat>
-            <v-container class="pa-4 pt-8">
-              <v-card-text id="scroll-target" data-test="scroll-area" class="scrollable-area">
-                <div v-scroll:#scroll-target="onScroll">
-                  <terms-of-use></terms-of-use>
-                </div>
-              </v-card-text>
-              <v-card-actions class="justify-center pb-4">
-                <v-btn
-                  large
-                  depressed
-                  color="primary"
-                  class="font-weight-bold mx-3 px-8"
-                  :disabled="!atBottom"
-                  @click="clickAccepted"
-                  data-test="accept-button"
-                >
-                  Accept Terms
-                </v-btn>
-                <v-btn
-                  large
-                  depressed
-                  color="primary"
-                  class="font-weight-bold mx-3 px-8"
-                  @click="clickDecline"
-                  data-test="decline-button"
-                >
-                  Decline
-                </v-btn>
-              </v-card-actions>
-            </v-container>
-          </v-card>
-        </v-col>
-      </v-row>
+
+    <div v-if="!isLoading">
+      <div class="view-header">
+        <h1>Terms of Use</h1>
+      </div>
+      <v-card class="profile-card" flat>
+        <v-card-text>
+          <terms-of-use></terms-of-use>
+        </v-card-text>
+        <v-card-actions class="terms-of-use-btns justify-center pt-0 pb-9">
+          <v-btn
+            large
+            color="primary"
+            class="font-weight-bold"
+            @click="clickAccepted"
```

Comment:
it seems like the disable property is removed here, should we have to enable the accept button even without reaching end of the line?

Reply:
I think the buttons will only be shown when the terms text has been scrolled (they are now placed in the same container, and we don't define a separate scroll area).  Correct me if I'm wrong @jeznorth

Reply:
This is a full page scroll now. Do we need this anymore? By default they need to scroll down to actually see the buttons.